editing/fx/passes/pact/harmonize.py

```diff
@@ -549,10 +536,10 @@ def rqs_merge_fun(gm : fx.GraphModule, match : Match):
     return RequantShift(newMul, newAdd, (rqs_2.n_levels_out)//(2**mixed), signed = signed, D=newD, cmsis_requant=rqs_1.cmsis_requant, requant_node=rqs_1.requant_node)
 
 class RQSMergePass(SequentialPass):
-    def __init__(self, **kwargs):
+    def __init__(self, symbolic_trace: callable = PACT_symbolic_trace, **kwargs):
```

Comment:
The type annotation looks wrong. I think it should be Callable[[Union[nn.Module, fx.GraphModule]], fx.GraphModule] or similar.